docs/reference/filters.md

```diff
@@ -2374,12 +2374,12 @@ Those metrics are not identifying the realm and client. They always have `*` in
 Example:
 
 ```
-                                                                                     + Realm
-                                                                                     |
+                                                                                      + Realm
```

Comment:
What this change is for?

Reply:
To make the annotations in doc point to the right place.
Before:
![Screenshot from 2023-03-06 17-40-05](https://user-images.githubusercontent.com/8851784/223175100-9098fb6d-0e89-41f3-b87e-fb2cc42a86e0.png)
After:
![Screenshot from 2023-03-06 17-40-11](https://user-images.githubusercontent.com/8851784/223175120-0b06c5a6-5c8b-4eb0-8e6e-a9af73d646ab.png)

Reply:
It looks ok to me 

https://github.com/zalando/skipper/blob/master/docs/reference/filters.md#available-metrics

![Screenshot from 2023-03-06 17-48-55](https://user-images.githubusercontent.com/697976/223176471-1d5d892e-e40d-40f1-abbb-c2adc8a2215c.png)

Reply:
Ah, right, should point to the star